liftwing/models/liftwing_model.py

```diff
@@ -10,7 +10,12 @@ def __init__(self, base_url: str, payload_model: Type[BaseModel] = None):
         self.base_url = base_url
         self.payload_model = payload_model
 
-    def request(self, payload: Dict[str, Any], method: str = "POST", headers: Dict[str, str] = None) -> Dict[str, Any]:
+    def request(
+        self,
+        payload: Dict[str, Any],
+        method: str = "POST",
+        headers: Dict[str, str] = None,
+    ) -> Dict[str, Any]:
         """
         This is an abstact method that makes a request to the API endpoint.
```

Comment:
```suggestion
        This is an abstract method that makes a request to the API endpoint.
```

Reply:
Removed it completely as it is no longer an abstract function.